src/main/java/io/appium/java_client/remote/MobilePlatform.java

```diff
@@ -22,4 +22,5 @@
     String IOS = "iOS";
     String FIREFOX_OS = "FirefoxOS";
     String WINDOWS = "Windows";
+    String TVOS = "tvOS";
```

Comment:
Does it make sense to have it inside `AutomationName` interface instead?

Reply:
No, I do not think so in this case.
It confuses users a bit since when we use tvOS, the capability should be like:

```
...
automationName: 'xcuites', # AutomationName.IOS_XCUI_TEST
platformName: 'tvOS'      # AutomationName.IOS_TVOS or MobilePlatform.TVOS (or APPLE_TVOS is proper?)
...
```

Reply:
Unlike youiengine tvos uses xcuitest as automation name. I still think having a TVOSdriver would also be helpful. That would avoid confusion for users around what capability needs to be used for tvos.

Reply:
I don't have a strong opinion about the new class.
I would respect @mykola-mokhnach, @saikrishna321 and @SrinivasanTarget

Reply:
I would say not to have TVOSDriver class. We should just document what capability to use for TVOS.